bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitTaggedItem.java

```diff
@@ -39,47 +40,55 @@
     public final static String STEP = "step";
 
     private static final Map<Integer, String> CREATED_ACCESSORY_IDS = new ConcurrentHashMap<>();
-    /**
-     * The type of HomekitDevice we've decided this was. If the item is question is the member of a group which is a
-     * HomekitDevice, then this is null.
-     */
-    private final Item item;
+
+    // proxy item used to group commands for complex item types like Color or Dimmer
+    private final HomekitOHItemProxy item;
```

Comment:
```suggestion
    private final HomekitOHItemProxy proxyItem;
```

I would suggest to rename this field as it is a bit confusing to have a field named `item` and call `.getItem()` on it to get the actual item.